src/models/linear_gaussian_conditionals.jl

```diff
@@ -1,16 +1,22 @@
 """
-    abstract type AbstractLGC end
+    AbstractLGC
 
 Represents a Gaussian conditional distribution:
+
 ```julia
-y | x ∼ Gaussian(A * x + a, Q)
+y | x ∼ Gaussian(A * xᵗ + a, Q)
```

Comment:
Why the `t` here?

Reply:
I first misunderstood the equation and badly reverted